libabyss/src/server.cpp

```diff
@@ -50,8 +56,11 @@ namespace abyss
         _conn->read   = &ConnImpl::OnRead;
         _conn->tick   = &ConnImpl::OnTick;
         _conn->closed = &ConnImpl::OnClosed;
-        m_Bad         = false;
-        m_State       = eReadHTTPMethodLine;
+#ifdef _WIN32
+        _conn->write = &TCPWrite;
+#endif
```

Comment:
this should be set by the event loop instead.

Reply:
ok